.github/workflows/vscode-ui.yaml

```diff
@@ -22,22 +22,13 @@ jobs:
           fetch-depth: 0
       - uses: ./.github/actions/setup
       - uses: extractions/setup-just@v2
-      - uses: actions/setup-node@v4
-        with:
-          node-version: 20.x
-      - uses: actions/checkout@v4
-        with:
-          repository: rstudio/connect-content
-          path: "test/content"
-          sparse-checkout: bundles
-          sparse-checkout-cone-mode: false
-          token: ${{ secrets.CONNECT_PAT }}
+
       - uses: actions/setup-python@v5
         with:
-          python-version: 3.9
-      - uses: quarto-dev/quarto-actions/setup@v2
-        with:
-          version: 1.4.549
+          python-version: ${{ env.PYTHON_VERSION }}
+      # - uses: quarto-dev/quarto-actions/setup@v2
+      #   with:
+      #     version: ${{ env.QUARTO_VERSION }}
```

Comment:
Can these be removed?

Reply:
I'll be adding Quarto tests later on so I'm just going to leave that there until it's ready.